lambdas/get-addresses/.eslintrc.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
     env: {
```

Comment:
When we get another typescript lambda, we should move this linting config one folder up.